easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0_data-dep-paths.patch

```diff
@@ -0,0 +1,113 @@
+pick up on $ALPHAFOLD_DATA_DIR to specify location to downloaded data
+pick up on HH-suite, HHMER, Kalign dependencies provided via EasyBuild
+author: Kenneth Hoste (HPC-UGent)
+diff --git a/run_alphafold.py b/run_alphafold.py
+index 61f7ef2..81c97a0 100644
+--- a/run_alphafold.py
++++ b/run_alphafold.py
+@@ -36,6 +36,45 @@ from alphafold.model import model
+ from alphafold.relax import relax
+ # Internal import (7716).
+ 
++
++data_dir = os.getenv('ALPHAFOLD_DATA_DIR')
++if data_dir:
++    uniref90_database_path = os.path.join(data_dir, 'uniref90')
++    mgnify_database_path = os.path.join(data_dir, 'mgnify')
++    bfd_database_path = os.path.join(data_dir, 'bfd')
++    uniclust30_database_path = os.path.join(data_dir, 'uniclust30')
++    pdb70_database_path = os.path.join(data_dir, 'pdb70')
++    template_mmcif_dir = os.path.join(data_dir, 'pdb_mmcif', 'mmcif_files')
++    obsolete_pdbs_path = os.path.join(data_dir, 'pdb_mmcif', 'obsolete.dat')
```

Comment:
```suggestion
+    uniref90_database_path = os.path.join(data_dir, 'uniref90', 'uniref90.fasta')
+    mgnify_database_path = os.path.join(data_dir, 'mgnify', 'mgy_clusters_2018_12.fa')
+    bfd_database_path = os.path.join(data_dir, 'bfd')
+    uniclust30_database_path = os.path.join(data_dir, 'uniclust30')
+    pdb70_database_path = os.path.join(data_dir, 'pdb70', 'pdb70')
+    template_mmcif_dir = os.path.join(data_dir, 'pdb_mmcif', 'mmcif_files')
+    obsolete_pdbs_path = os.path.join(data_dir, 'pdb_mmcif', 'obsolete.dat')
```

I also had the following instead of `bfd_database_path`:
```
small_bfd_database_path=data/small_bfd/bfd-first_non_consensus_sequences.fasta
```

Reply:
`small_bfd_database_path` is unused in AlphaFold 2.0.0, that was only added later (see https://github.com/deepmind/alphafold/commit/0bab1bf84d9d887aba5cfb6d09af1e8c3ecbc408)

Reply:
I don't have a `mgy_clusters_2018_12.fa` in `mgnify`, only `mgy_clusters.fa`.

This was changed after the 2.0.0 release, so `mgy_clusters.fa` is correct for this version, that's also the filed you'll get when downloading the data with the script included in installation directory of 2.0.0...